src/ir/builtins.py

```diff
@@ -111,6 +111,27 @@ def get_number_types(self):
             self.get_big_integer_type(),
         ]
 
+    def get_decl_candidates(self, gen_object):
+        """ Overwrite this method to return a list
+        with language-specific AST declaration nodes.
+
+        See implementation in typescript_types.py and
+        TS-specific AST nodes in typescript_ast.py
+
+        """
+        return []
+
+    def update_add_node_to_parent(self, gen_object):
+        """ Overwrite this to update the dict 'node_type'
+        on src.generators.generator._add_node_to_parent
+        with the respective <ast.Node, src.ir.context>
+        key-value pair for the language-specific AST nodes.
+
+        See implementation in typescript_types.py
+
+        """
+        return {}
```

Comment:
Should this method have a return value? If not, use `pass` instead.

Reply:
Yes, it should return a value.